src/components/TrackGameTwo.js

```diff
@@ -0,0 +1,157 @@
+import styled from 'styled-components';
+import Input from './Input.js';
+import PropTypes from 'prop-types';
+
+TrackGameTwo.propTypes = {
+  onHandleChange: PropTypes.func,
+  gameData: PropTypes.object
+};
+
+export default function TrackGameTwo({ onHandleChange, gameData }) {
```

Comment:
The code in this component is rather redundant. 🤔 Maybe think about changing the data to an array?
```js
const gameDataArray = [
{ name: "playerNameOne", value: gameData.playerNameOne, score: ...},
{ name: "playerNameTwo", value: ..., score: ...},
{ name: "playerNameThree", value: ..., score: ...},
{ name: "playerNameFour", value: ..., score: ...},
]
```
You could even do this here, I think, then you wouldn't need to update all of your data structure. Maybe 😅  Not completely sure but it seems a bit DRY. 😉